src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx

```diff
@@ -174,18 +174,14 @@ Our .NET agent relies on environment variables to tell the .NET Common Language
     ```
     CORECLR_ENABLE_PROFILING=1
     CORECLR_PROFILER={36032161-FFC0-4B61-B559-F6C5D41BAE5A}
-    NEWRELIC_INSTALL_PATH=<var>path\to\agent\directory</var>
-    CORECLR_NEWRELIC_HOME=<var>path\to\agent\directory</var>
+    NEWRELIC_INSTALL_PATH=<var>%PROGRAMFILES%\New Relic\.NET Agent\</var>
```

Comment:
I think we should actually use `C:\Program Files` here since if a customer puts %PROGRAMFILES% in and it doesn't do the ENV VAR expansion during the set operation, the agent will not work properly. Also, it will not match what the installer sets.

I don't think our basic documentation should worry about users that have set a non-default program files location